src/components/shop-cart/input/label-input-aside.component.spec.ts

```diff
@@ -18,4 +22,64 @@ describe('LabelInputAsideComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
-});
+
+
+  it('should update card number error message', () => {
+    component.cardNumberEntry.setValue('invalidCardNumber');
+    expect(component.getCardNumberErrorMessage()).toContain('wrong format');
```

Comment:
I think the message could be improved, something like: "Invalid Card Number Wrong format", because use toContain could be ambiguous